packages/app/ui/skeletons/BeatSkeleton.tsx

```diff
@@ -33,5 +33,13 @@ export function BeatSkeleton() {
     );
   };
 
-  return <>{Array(8).fill(<EntrySkeleton />)}</>;
+  return (
+    <>
+      {Array(8)
+        .fill(true)
+        .map((_, i) => (
+          <EntrySkeleton key={i} />
```

Comment:
What is the reason for this change?

Reply:
Each element in the list should have an unique key. I missed that in the previous pr.